pom.xml

```
@@ -95,6 +96,11 @@
             <artifactId>jackson2-api</artifactId>
             <version>${jackson2-api.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>bouncycastle-api</artifactId>
+            <version>${bouncycastle-api.version}</version>
+        </dependency>
```

```suggestion
            <scope>runtime</scope>
        </dependency>
```

as a reminder that we need not compile against it.